webClient/package.json

```diff
@@ -9,8 +9,7 @@
     "test": "exit 0"
   },
   "dependencies": {
-    "@angular/cdk": "~6.3.3",
-    "angular-tree-component": "~7.2.0",
```

Comment:
Great find here that we don't need it. A relic of a bygone era...

Reply:
Exactly! :)